frontend/packages/noobaa-storage-plugin/src/components/create-backingstore-page/create-bs.tsx

```diff
@@ -198,6 +198,12 @@ const PVCType: React.FC<PVCTypeProps> = ({ state, dispatch }) => {
     TiB: 'TiB',
   };
 
+  // Noobaa expected Ti console standrad is to show TiB
+  const unitConverter = {
+    GiB: 'Gi',
+    TiB: 'Ti',
+  };
```

Comment:
This should be moved to constants file.

Reply:
This is already merged. See the .. diff.